core/src/main/scala/kafka/log/AbstractIndex.scala

```diff
@@ -89,7 +89,7 @@ abstract class AbstractIndex(@volatile private var _file: File, val baseOffset:
       lookup. So that, the entire warm section is really "warm".
       When doing warm-section lookup, following 3 entries are always touched: indexEntry(end), indexEntry(end-N),
       and indexEntry((end*2 -N)/2). If page size >= 4096, all the warm-section pages (3 or fewer) are touched, when we
-      touch those 3 entries. As of 2018, 4096 is the smallest page size for all the processors (x86-32, x86-64, MIPS,
+      touch those 3 entries. As of 2048, 4096 is the smallest page size for all the processors (x86-32, x86-64, MIPS,
```

Comment:
This change doesn't make sense. `2018` was meant to be the year.

Reply:
will revert it. sorry for rough merge :(

Reply:
#9751